addons/account/models/chart_template.py

```diff
@@ -223,11 +223,13 @@ def _load(self, template_code, company, install_demo, force_create=True ):
         if reload_template:
             self._pre_reload_data(company, template_data, data, force_create)
             install_demo = False
-        data = self._pre_load_data(template_code, company, template_data, data)
-        self._load_data(data)
-        self._post_load_data(template_code, company, template_data)
-        self._load_translations(companies=company)
+        # allow reloading the data only if we force create it
```

Comment:
I don't understand this; it is wrong to me.

Reply:
I have modified it the goal was to prevent changing the default values in the settings while upgrading